src/Command/CleanMediaCommand.php

```diff
@@ -29,35 +28,23 @@
 /**
  * @final since sonata-project/media-bundle 3.21.0
  */
-class CleanMediaCommand extends Command
+class CleanMediaCommand extends BaseCommand
```

Comment:
This change should not be done

Reply:
https://github.com/sonata-project/SonataMediaBundle/pull/1765#discussion_r457467169

Reply:
Why to change a class that is already working....

Reply:
Please revert this, this class was already fine